src/main/java/io/kokuwa/maven/helm/pojo/Dependencies.java

```diff
@@ -0,0 +1,23 @@
+package io.kokuwa.maven.helm.pojo;
+
+import java.util.ArrayList;
+
+import lombok.Data;
+
+/**
+ * POJO for list of Chart.yaml dependencies
+ *
+ * @since 6.9.1
+ */
+@Data
+public class Dependencies {
+	private ArrayList<Dependency> dependencies;
+
+	@Data
+	public static class Dependency {
+		private String name;
+		private String version;
+		private String repository;
+		private boolean overwrite = false;		
```

Comment:
A: Worth some docs on this property since it's not inherent to the dependency in the YAML and is more about internal record keeping for the plugin